packages/accounts/src/light-account/account.ts

```diff
@@ -32,45 +37,72 @@ import {
 } from "./utils.js";
 
 export type LightAccount<
+  TEntryPointVersion extends EntryPointVersion,
   TSigner extends SmartAccountSigner = SmartAccountSigner
-> = SmartContractAccountWithSigner<"LightAccount", TSigner> & {
+> = SmartContractAccountWithSigner<
+  TEntryPointVersion,
+  "LightAccount",
+  TSigner
+> & {
   getLightAccountVersion: () => Promise<LightAccountVersion>;
   encodeTransferOwnership: (newOwner: Address) => Hex;
   getOwnerAddress: () => Promise<Address>;
 };
 
 export type CreateLightAccountParams<
+  TEntryPointVersion extends EntryPointVersion,
   TTransport extends Transport = Transport,
   TSigner extends SmartAccountSigner = SmartAccountSigner
 > = Pick<
-  ToSmartContractAccountParams<"LightAccount", TTransport>,
-  "transport" | "chain" | "entryPoint" | "accountAddress"
+  ToSmartContractAccountParams<TEntryPointVersion, "LightAccount", TTransport>,
+  "transport" | "chain"
```

Comment:
wait, why remove the `accountAddress` from this pick?

Reply:
just checked it locally and I'm not seeing an error by keeping it

Reply:
why was this resolved? mind adding reasons when resolving comments? it's helpful to have more context. if there's an error it's helpful to include a screenshot of the error or console dump of the error

Reply:
bump

Reply:
ah missed. was actually done on the child PR, but as I refactored fixes into here, that reverted .